frontend/packages/dev-console/src/components/helm/HelmReleaseDetailsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router';
+import {
+  navFactory,
+  ResourceSummary,
+  SectionHeading,
+  LoadingBox,
+  StatusBox,
+  FirehoseResult,
+} from '@console/internal/components/utils';
+import { SecretModel } from '@console/internal/models';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { DetailsPage } from '@console/internal/components/factory';
+import { K8sResourceKindReference } from '../../../../../public/module/k8s';
+
+const SecretReference: K8sResourceKindReference = 'Secret';
+const HelmReleaseReference = 'HelmRelease';
+
+export interface HelmReleaseDetailsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+  secrets?: FirehoseResult;
+}
+
+export const HelmReleaseDetails: React.SFC<any> = ({ obj: resourceDetails }) => {
+  const helmRelease = resourceDetails;
+  helmRelease.metadata.name = resourceDetails.metadata.labels.name;
```

Comment:
will `metadata.labels` always be defined  if  not  will be good to have a check

Reply:
yes, because I am looking for a particular secret resource having the label: name=<release-name>. If you see in `HelmReleasesDetailsPage.tsx` I have added `selector: { name: `${helmReleaseName}` }` in the firehose resources